e2e-tests/playwright/tests/functional/system_console/system_users/actions.spec.ts

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {TestBrowser} from '@e2e-support//browser_context';
+import {Client, createRandomTeam, createRandomUser} from '@e2e-support/server';
+import {expect, test} from '@e2e-support/test_fixture';
+import {components} from '@e2e-support/ui/components';
+import {SystemConsolePage} from '@e2e-support/ui/pages/system_console';
+import {getRandomId} from '@e2e-support/util';
+import {UserProfile} from '@mattermost/types/users';
+
+/**
+ * Setup a new random user, and search for it such that it's the first row in the list
+ * @param pw
+ * @param pages
+ * @returns A function to get the refreshed user, and the System Console page for navigation
+ */
+async function setupAndGetRandomUser(
```

Comment:
can this function just create Admin user and return. Omitting the search and verify? Search.spec.ts covers the searching tests for us.

Reply:
Yes so the tests should always be stateless and it makes sure that we always have a random user in our test and if you observe the function in the last it makes sure that particular user is in the top row. Thus making it easy for test to open its actions menu